server/application/application.go

```diff
@@ -357,6 +357,10 @@ func (s *Server) Patch(ctx context.Context, q *application.ApplicationPatchReque
 		return nil, err
 	}
 
+	if err = s.enf.EnforceErr(ctx.Value("claims"), rbacpolicy.ResourceApplications, rbacpolicy.ActionOverride, appRBACName(*app)); err != nil {
```

Comment:
Please not that this currently adds a requirement for the `override` permission to patch a resource. **This is likely to change current functionality.**